requirements/specifications/openrpc-extensions/app-passthrough-apis.md

```diff
@@ -108,15 +108,19 @@ Firebolt OpenRPC **MUST** support a `string` `x-provider-hint-parameter` extensi
 }
 ```
 
-The parameter denoted by `x-provider-hint-parameter` **MUST** exist and be a string.
+The parameter denoted by `x-provider-selection` **MUST** exist and be a string.
```

Comment:
Think this sentence is no longer needed? As if x-provider-selection is "focus" then it doesn't need to have a corresponding parameter named "focus"